### ISSUE TITLE
Elasticsearch SSL

### ISSUE THREAD

**Issue by Madhu1512** [mozilla-services/lua_sandbox_extensions#152]
Is there a plan to add SSL Support for elasticsearch output?

**Comment by trink** [comment]
No, not internally as we don't use ES; I will flag it for community support and see if it gets any interest.
